plugin/src/main/kotlin/org/jlleitschuh/gradle/ktlint/KtlintPlugin.kt

```diff
@@ -273,9 +273,14 @@ open class KtlintPlugin : Plugin<Project> {
         android.set(extension.android)
         ignoreFailures.set(extension.ignoreFailures)
         outputToConsole.set(extension.outputToConsole)
-        if (!target.isConsolePlain()) {
-            coloredOutput.set(extension.coloredOutput)
-        }
+        coloredOutput.set(extension.coloredOutput.map {
+            if (target.isConsolePlain()) {
+                target.logger.info("Console type is plain: disabling colored output")
```

Comment:
`info` isn't logged by default correct? If so 👍

Reply:
Yes, by default warn and error is logged.